pkg/configuration/configuration.go

```diff
@@ -139,6 +139,10 @@ const (
 	// varVerificationExcludedEmailDomains contains a comma-separated list of domain names for which phone verification
 	// is not required.  For example: "redhat.com,ibm.com"
 	varVerificationExcludedEmailDomains = "verification.excluded_email_domains"
+
+	// varVerificationExpiry is used to set the amount of time a verification code is active for before expiring
```

Comment:
Amount of time? In what units? Minutes I guess?
Let's call it `verification.code_expires_in_min` so it's clear what this is about.

Reply:
sorry, i added this in the Get func comment but guess i forgot to add it here.. anyways i've updated it here: 97cfceeac77f1fb7ff25e3b81945a35b07093d4d and 873f3c16c9e09e21ab31ef29e475c9e2ca115d14